internal/alertmanager/currentalert/process.go

```diff
@@ -91,11 +94,17 @@ func (e *examiner) processAlert(ds disableScaling) (bool, error) {
 
 	switch e.Alert.Annotations["command"] {
 	case "addPVC":
+		if err := checkLabelExists(e.Alert.Labels, "persistentvolumeclaim"); err != nil {
+			return false, emperror.Wrap(err, "label: persistentvolumeclaim")
+		}
 		err := addPVC(e.Alert.Labels, e.Alert.Annotations, e.Client)
 		if err != nil {
 			return false, err
 		}
 	case "downScale":
+		if err := checkLabelExists(e.Alert.Labels, "kafka_cr"); err != nil {
```

Comment:
I'd introduce types for alerts and define a validation function which could be invoked after filtering.